go/host/l1/publisher.go

```diff
@@ -77,8 +80,8 @@ func NewL1Publisher(
 		blobResolver:              blobResolver,
 		hostStopper:               hostStopper,
 		logger:                    logger,
-		maxWaitForL1Receipt:       maxWaitForL1Receipt,
-		retryIntervalForL1Receipt: retryIntervalForL1Receipt,
+		maxWaitForL1Receipt:       4 * time.Minute,
```

Comment:
is this temporary?

Reply:
Yes, just to test if our issues will disappear if we use optimism's wait for blob txes (tho this will be for all, but let's see how it behaves)

Reply:
why not change the config?

Reply:
its configured based on l1 block time

Reply:
6*l1 block time